requestor/request.go

```diff
@@ -1,4 +1,4 @@
-package server
+package requestor
```

Comment:
`requestor` name is confusing, since there is no object `requestor` here - it's just a bunch of functions.
What about package `request` or `requests`?
I.e. `requests.NewGetRequest` looks quite good.

Reply:
requests.Get looks even better :)